_ext/Pechakucha/setup.py

```diff
@@ -1,10 +1,7 @@
 from setuptools import setup, find_packages
 
 setup(
-    name='sphinxcontrib-pechakucha',
+    name='pechakucha',
     version='0.1',
     packages=find_packages(),
-    install_requires=[
-        'Sphinx',
-    ],
```

Comment:
I think this is still true; sphinx is a dependency to be able to use your extension

Reply:
im kinda confused on what this statement means

Reply:
you deleted sphinx as a dependency, which doesn't make sense to me